### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by lawmanuk** [kkrruumm/void-install-script#2]
Hi

Thanks for this amazing script

Is it possible to consider the following additions

1.  Btrfs 
2. Installation on gpt partition without deleting rest of drive

Thanks

**Comment by kkrruumm** [comment]
Hey,

1. Yeah, btrfs is an option I've been looking into a bit. I'll for sure be adding it at some point shortly enough, though at the moment I'm currently in the process of transitioning this installer to a TUI which will make it a bit easier to organize options as more and more get added without making the installer feel "weighted down" by the amount of stuff.

2. For sure, I can add an option to select partitions that already exist manually.

**Comment by amethystgosling163** [comment]
Hi @kkrruumm . I'd like to ask, what btrfs subvolumes does this installer create and are they snapper compatible?

**Comment by kkrruumm** [comment]
@amethystgosling163 by default,

root is mounted at @ 
snapshots is mounted at @snapshots (/.snapshots)
if you choose to split off home, the volume will be @home, of course mounted at /home

other subvols would include:
/var/cache/xbps
/var/tmp
/srv

because generally one would not include these in snapshots

i've not used snapper, but snapshotting should work as it does on other things that deploy btrfs

**Comment by amethystgosling163** [comment]
Thanks for answering!

**Comment by kkrruumm** [comment]
> Thanks for answering! Can you put /var/log on a seperate subvolume, by default, called @var_log, so that one can boot into snapper snapshots from grub-btrfs w/o converting having to convert the snapshots into writable ones?

yeah i'll look into it, i don't use btrfs myself so i appreciate suggestions

..adding grub-btrfs and snapper as options could also be cool.
